subcerts_test.go

```diff
@@ -252,7 +256,9 @@ var dcTesters = []struct {
 	name             string
 }{
 	{true, true, false, VersionTLS12, VersionTLS12, 0, "tls12", true, true, "tls12"},
-	{true, true, false, VersionTLS13, VersionTLS13, 0, "tls13", true, true, "tls13"},
+	// TODO: DC tests do not work with TLS 1.3
+	//       this is to be fixed by GH#108
+	// {true, true, false, VersionTLS13, VersionTLS13, 0, "tls13", true, true, "tls13"},
```

Comment:
For Chris to provide with a fix

Reply:
You will also need #108, since this PR drops support for DCs with TLS 1.2, as per draft-02 of the spec. The test cases have also changed.

Reply:
nop, I test data only